locations/storefinders/rio_seo.py

```diff
@@ -20,16 +21,20 @@ class RioSeoSpider(Spider):
         `/api/getAsyncLocations` and including parameters similar to
         `?template=search&level=search`
       - `template`: mandatory parameter, should be either "domain" or "search"
-      - `radius`: optional parameter, default value is 20038
-      - `limit`: optional parameter, default valus is 3000
+      - `radius`: optional parameter, default value is 10000
+      - `limit`: optional parameter, default valus is 10000
     """
 
     dataset_attributes = {"source": "api", "api": "rio_seo"}
-
-    end_point: Optional[str] = None
+    end_point: str = None
     limit: int = 10000
-    radius: int = 20038
+    radius: int = 10000
```

Comment:
With this conflict, I have chosen 20038 as that's what the other rioseo PR chose - will swapping this back to 10000 have a reduction in data collected?